### PR TITLE
BXMSPROD-1250 GIT_INFORMATION_HASHES

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1250

- https://github.com/kiegroup/kogito-pipelines/pull/137
- https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/148

### CODE REVIEW

File: test/vars/UtilSpec.groovy [hunk 1]
```diff
@@ -464,9 +466,11 @@ class UtilSpec extends JenkinsPipelineSpecification {
         groovyScript.storeGitInformation('projectName')
         then:
         1 * getPipelineMock('githubscm.getCommit')() >> 'kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102)'
+        1 * getPipelineMock('githubscm.getCommitHash')() >> '11111111111111111111111111111111'
         1 * getPipelineMock('githubscm.getBranch')() >> '* (detached from 0f917d4)  remotes/origin/master'
         1 * getPipelineMock('githubscm.getRemoteInfo')('origin', 'url') >> 'https://github.com/kiegroup/lienzo-core.git'
         env['GIT_INFORMATION_REPORT'] == 'projectName=kiegroup/lienzo-tests: 45c16e1 Fix tests (#84) Branch [* (detached from 45c16e1)  remotes/origin/master] Remote [https://github.com/kiegroup/lienzo-tests.git]; projectName=kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102) Branch [* (detached from 0f917d4)  remotes/origin/master] Remote [https://github.com/kiegroup/lienzo-core.git]'
+        env['GIT_INFORMATION_HASHES'] == "projectName=11111111111111111111111111111111"
```
Comment (radtriste) [inline test/vars/UtilSpec.groovy:473]: I guess here you should test with a previous value stored in `env['GIT_INFORMATION_HASHES']` as it is done for `env['GIT_INFORMATION_REPORT']`. wdyt ?

File: test/vars/UtilSpec.groovy [hunk 2]
```diff
@@ -449,9 +449,11 @@ class UtilSpec extends JenkinsPipelineSpecification {
...
         then:
         1 * getPipelineMock('githubscm.getCommit')() >> 'kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102)'
+        1 * getPipelineMock('githubscm.getCommitHash')() >> 'ac36137f12d1bcfa5cdf02b796a1a33d251b48e1'
         1 * getPipelineMock('githubscm.getBranch')() >> '* (detached from 0f917d4)  remotes/origin/master'
         1 * getPipelineMock('githubscm.getRemoteInfo')('origin', 'url') >> 'https://github.com/kiegroup/lienzo-core.git'
         env['GIT_INFORMATION_REPORT'] == "projectName=kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102) Branch [* (detached from 0f917d4)  remotes/origin/master] Remote [https://github.com/kiegroup/lienzo-core.git]"
+        env['GIT_INFORMATION_HASHES'] == "projectName=ac36137f12d1bcfa5cdf02b796a1a33d251b48e1"
     }
 
     def "[util.groovy] storeGitInformation GIT_INFORMATION_REPORT previous value"() {
```
Comment (radtriste) [inline test/vars/UtilSpec.groovy:459]:
```suggestion
    def "[util.groovy] storeGitInformation with previous values"() {
```

File: test/vars/UtilSpec.groovy [hunk 3]
```diff
@@ -449,9 +449,11 @@ class UtilSpec extends JenkinsPipelineSpecification {
         groovyScript.storeGitInformation('projectName')
         then:
         1 * getPipelineMock('githubscm.getCommit')() >> 'kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102)'
+        1 * getPipelineMock('githubscm.getCommitHash')() >> 'ac36137f12d1bcfa5cdf02b796a1a33d251b48e1'
         1 * getPipelineMock('githubscm.getBranch')() >> '* (detached from 0f917d4)  remotes/origin/master'
         1 * getPipelineMock('githubscm.getRemoteInfo')('origin', 'url') >> 'https://github.com/kiegroup/lienzo-core.git'
         env['GIT_INFORMATION_REPORT'] == "projectName=kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102) Branch [* (detached from 0f917d4)  remotes/origin/master] Remote [https://github.com/kiegroup/lienzo-core.git]"
+        env['GIT_INFORMATION_HASHES'] == "projectName=ac36137f12d1bcfa5cdf02b796a1a33d251b48e1"
```
Comment (radtriste) [inline test/vars/UtilSpec.groovy:456]: rename test method `storeGitInformation no previous values`

**Review by radtriste — CHANGES_REQUESTED** [review]

File: test/vars/UtilSpec.groovy (hunk 3 above)
Comment (Ginxo) [reply test/vars/UtilSpec.groovy:456]: done

File: test/vars/UtilSpec.groovy (hunk 1 above)
Comment (Ginxo) [reply test/vars/UtilSpec.groovy]: you are totally right, done!

**Comment by Ginxo** [comment]
@radtriste thanks for your review. Changes applied.

**Review by radtriste — APPROVED** [review]

**Review by mbiarnes — APPROVED** [review]
LGTM

**Pull request merged**